File: Python/Exercicio18.py
# Algoritmo que determina o maior entre n números sendo que a condição de parada é um valor 0.

# Variável
maior = 0

# Entrada
numero = int(input("Escreva um número: "))

# Processamento
while numero != 0:
    if numero > maior:
        maior = numero
    numero = int(input("Escreva um número: "))
print("O maior número é o {0}".format(maior))

File: C/Exercicio18.c
//Algoritmo que determina qual é o maior número dentre os números inseridos.
//A condição de parada é a entrada de um número 0.
#include <stdio.h>

int main() {
    //Variáveis
    int n, maior = 0;

    //Entrada
        printf("Digite um número: \n");
        scanf("%d", &n);

    //Processamento
    while (n != 0) {
        if (n > maior) {
            maior = n;            
        }
        //Entrada
        printf("Digite um número: \n");
        scanf("%d", &n);
    }
    printf("Maior número: %d \n", maior);
    return 0;
}